docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/ws/MappedFsWorkspaceBrowser.java

```diff
@@ -25,17 +29,13 @@ public FilePath getWorkspace(Job job) {
         Run lastBuild = job.getLastBuild();
 
         if (lastBuild != null) {
-            DockerBuildAction a = lastBuild.getAction(DockerBuildAction.class);
+            DockerBuildAction action = lastBuild.getAction(DockerBuildAction.class);
 
-            if (a != null && !Strings.isNullOrEmpty(a.remoteFsMapping)) {
-                File mappedRemoteWorkspace = new File(a.remoteFsMapping);
-                mappedRemoteWorkspace = new File(mappedRemoteWorkspace, "workspace");
-                mappedRemoteWorkspace = new File(mappedRemoteWorkspace, job.getName());
```

Comment:
Construction of full path - `fsMapping/workspace/jobname` with help of `File(File parent, String child)`. Same as inside of `getFile` method in commons-io